maquetado/scss/tech.scss

```diff
@@ -27,18 +27,28 @@
     padding: 10px;
     position: relative;
     @include card-size();
-    
+
     &::after {
         content: '';
         position: absolute;
         border: solid 1px $green;
         border-radius: 8px;
-        top: -11px;
-        left: -11px;
+        top: -10px;
+        left: -10px;
         z-index: -1;
         @include card-size();
     }
 
+    &:hover:after {
+        content: '';
+        position: absolute;
+        border: solid 1px $green;
```

Comment:
If im not mistaken, there is no need to repeat every attribute that you defined in the :after pseudo element. You can only define the attributes that must change on hover. In this case, it would be `top`, `left` and `transition`

Reply:
ok, I'll remove it.